src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/PreferIsKindAnalyzerTests.cs

```diff
@@ -419,5 +419,63 @@ End Class
 
             await VerifyVB.VerifyCodeFixAsync(source, source);
         }
+
+        [Fact]
+        public async Task TestSingleNullConditionalAccess_CS()
+        {
+            var source =
+@"using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+class C
+{
+    bool Method(SyntaxNode node)
```

Comment:
💡 Can you add a test for `SyntaxToken?` as well?

Reply:
@sharwell The codefix (currently) produces:

```csharp
    bool Method(SyntaxToken? token)
    {
        return token.IsKind(SyntaxKind.None); // <--  error CS1929: 'SyntaxToken?' does not contain a definition for 'IsKind' and the best extension method overload 'CSharpExtensions.IsKind(SyntaxToken, SyntaxKind)' requires a receiver of type 'SyntaxToken'
    }
```

Going to look how I'll fix that.

Reply:
I found it easier to not report a diagnostic rather than complicating the logic inside the codefix. The usage of `Nullable<SyntaxToken>` is a little bit rare. But let me know if you want me to go the other route and make the codefix behave correctly.

Reply:
Skipping it is what I expected. Totally fine.